src/Register.php

```diff
@@ -0,0 +1,73 @@
+<?php namespace Leean\Widgets;
+
+/**
+ * Class to provide widget functions.
+ */
+class Register
+{
+	/**
+	 * Registered widgets for this project.
+	 *
+	 * @var array
+	 */
+	private static $_registered_widgets = [];
+
+	/**
+	 * Init. Takes $widgets_to_register args in the following format:
+	 * [
+	 * 	 'leean' => [ 'LeeanPreview', 'LeeanMenu' ],
+	 *   'custom' => [ 'MyNamespace\MyWidget' ],
+	 * ]
+	 *
+	 * @param array $widgets_to_register Set of widgets to register for your project.
+	 */
+	public static function init( $widgets_to_register = [] ) {
+		$widgets_to_register = wp_parse_args( $widgets_to_register, [
+			'leean' => [],
+			'custom' => [],
+		] );
+
+		self::$_registered_widgets = $widgets_to_register;
+
+		add_action( 'widgets_init', [ __CLASS__, 'unregister_all' ], 11 );
```

Comment:
I think methods should be named inversed like: 

 - start / stop
- register / unregister